controlpanel/api/models/tool.py

```diff
@@ -73,11 +80,12 @@ class ToolDeployment:
 
     objects = ToolDeploymentManager()
 
-    def __init__(self, tool, user):
+    def __init__(self, tool, user, outdated=False):
         self._id_token = None
         self._subprocess = None
         self.tool = tool
         self.user = user
+        self.outdated = outdated
```

Comment:
I think `outdated` would be better as a property or `is_outdated` method on `ToolDeployment`. Maybe pass in the `deployed_version` to the constructor instead?

Reply:
I'm not keen on passing the `deployed_version` as I don't want to make this mandatory and when I make it optional the semantic of this gets weird: Does `deployed_version=None` means not deployed or out of date? I don't want to get into that as can lead to confusion/bugs.

But I like the idea of just having a `is_outdated` property which we set in the `ToolDeploymentManager` - it may not be the best solution but it would work for the time being.